Lib/logging/__init__.py

```diff
@@ -1120,6 +1120,8 @@ def setStream(self, stream):
     def __repr__(self):
         level = getLevelName(self.level)
         name = getattr(self.stream, 'name', '')
+        if not isinstance(name, str):
```

Comment:
I would suggest not bothering with the `if` - just call `str()` unconditionally, but add a comment referencing the issue so that the reason for the call is clear.

Reply:
Updated, thanks for the review